charts/portworx/templates/storage-cluster.yaml

```diff
@@ -168,13 +179,32 @@ spec:
     enabled: false
     {{- end }}
 
-  {{- if eq $isCoreOS true}}
+  {{- if eq $volumesPresent "true" }}
   volumes:
-    - name: src
-      mountPath: /lib/modules
-      hostPath:
-        path: /lib/modules
-        type: Directory
+  {{- if eq $isCoreOS true}}
+  - name: src
+    mountPath: /lib/modules
+    hostPath:
+      path: /lib/modules
+      type: Directory
+  {{- end }}
+  {{- if ne $etcdCertPath "none" }}
+  - name: etcdcerts
+    mountPath: /etc/pwx/etcdcerts
+    secret:
+      secretName: px-etcd-certs
+      items:
+      - key: ca.pem
```

Comment:
This is to keep backward compatibility for existing clusters. We don't want to change the name of the files in the middle of a migration. 
This is how the files were named before - https://github.com/portworx/helm/blob/master/charts/portworx/templates/portworx-ds.yaml#L363

Also, for new clusters we should not recommend using it this way. They should directly pass the secret name.

Reply:
I'm not sure if it's good to add an extra volume here named `etcdcerts` and have the same mountPath as default operator one. In this case we need to make sure the new kvdb.authSecret is not configured. otherwise with conflict mount path those volumes will be ignored.

Reply:
oh you're right, etcdcerts and kvdbcerts,